ci/jenkins/test-vm.sh

```diff
@@ -231,7 +287,8 @@ function run_e2e_vms {
 
 function deliver_antrea_vm {
     export_govc_env_var
-    clean_vm_agent
+    clean_vm_agent ${!LINUX_HOSTS_TO_IP[@]}
```

Comment:
In this we can directly use the host names. But the other places we are mapping the IP address with the host name. For example, install_on_linux method we require host name and IP address. The IP address is use to login and host name is use to fetch the username of that host.

Reply:
Even in install_on_linux, I don't get why it can't loop the existing LIN_HOSTNAMES, but need to extract indices from LINUX_HOSTS_TO_IP. You are getting IP and username by querying the other maps anyway, why looping LIN_HOSTNAMES affects it?

Reply:
Because how do we know the username and IP address of that host name.

Reply:
I don't know how they are related. This is only about how the host name is got, has nothing to do with the loop body. You are suggesting the code below can't work?
```
function install_on_linux {
   for host_name in "${LIN_HOSTNAMES[@]}"; do
      echo "Installing on Linux VM $host_name"
      USERNAME=${LINUX_HOSTS_TO_USERNAME[${host_name}]}
      IP_ADDRESS=${LINUX_HOSTS_TO_IP[${host_name}]}
      ssh -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" -n ${USERNAME}@${IP_ADDRESS} "sudo chmod +x /tmp/antrea-ci/install-vm.sh"
      ssh -o StrictHostKeyChecking=no -i "${WORKDIR}/jenkins_id_rsa" -n ${USERNAME}@${IP_ADDRESS} "cd /tmp/antrea-ci && sudo ./install-vm.sh --ns vm-ns --bin ./antrea-agent --config ./antrea-agent.conf --kubeconfig ./antrea-agent.kubeconfig  --antrea-kubeconfig ./antrea-agent.antrea.kubeconfig"
   done
}
```
Not sure if @Anandkumar26 can get my point.

Reply:
Actually understood. Done. Thanks.